packages/squiggle-lang/src/rescript/Distributions/GenericDist.res

```diff
@@ -159,6 +119,89 @@ let toPointSet = (
   }
 }
 
+module Score = {
+  type scoreDistOrScalar = DistributionTypes.DistributionOperation.scoreDistOrScalar
+
+  type pointSet_ScoreDistOrScalar = D(PointSetTypes.pointSetDist) | S(float)
+  let argsMake = (
+    ~esti: scoreDistOrScalar,
+    ~answ: scoreDistOrScalar,
+    ~prior: option<scoreDistOrScalar>,
+  ): result<PointSetDist_Scoring.scoreArgs, error> => {
+    let toPointSetFn = toPointSet(
+      ~xyPointLength=MagicNumbers.Environment.defaultXYPointLength,
+      ~sampleCount=MagicNumbers.Environment.defaultSampleCount,
```

Comment:
Shouldn't this take ``env`` for this?

Reply:
I was following the signature of the code as I found it--- but I can certainly go ahead and modify

Reply:
https://github.com/quantified-uncertainty/squiggle/issues/718

Reply:
I'd like not to piggyback this onto the current story, so I filed a new story.